var/spack/repos/builtin/packages/py-pytest-httpbin/package.py

```diff
@@ -0,0 +1,44 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyPytestHttpbin(PythonPackage):
+    """Easily test your HTTP library against a local copy of httpbin"""
+
+    homepage = "https://github.com/kevin1024/pytest-httpbin"
+    url      = "https://github.com/kevin1024/pytest-httpbin/archive/v0.2.0.tar.gz"
+
+    version('0.2.3', 'b8ebb8e2fbac1a445fb5d044f7fec556',
+        url='https://pypi.python.org/packages/12/12/5430600cb9417080b561237761ff2dffde520b664cc352433d2e57051222/pytest-httpbin-0.2.3.tar.gz')
+
+    extends('python', ignore=r'bin/flask')
```

Comment:
Can you please add a comment explaining this `ignore=` clause?  It's not commonly used in Spack, and deserves a bit of explanation.